lab/types.py

```diff
@@ -164,8 +190,6 @@ def _name_to_numpy_dtype(name):
         return int
     elif name == "bool":
         return bool
-    elif name == "unicode":
-        return str
```

Comment:
On older versions of dependencies, this runs into problems because it might attempt to get `np.unicode`, which doesn't exist. Did you have to remove these lines because it caused an issue? Would it be possible to keep them in?

Reply:
Ah, I had to remove these as the coveralls complained about reduced coverage after I dropped python-3.8. This is because on the later Pythons, numpy-2 is installed and tested against (and has no `unicode`). Naturally, I forgot to also bump the required numpy version in the `setup.py`. Unless you'd like to keep supporting the old numpy, I can update the requirements to `numpy>=2.0` ?

Reply:
Of course!